docs/jupyter/plotly.ipynb

```diff
@@ -0,0 +1,6597 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Testing Plotly Modin Interoperability"
```

Comment:
I would say smth like "Demonstrating" not "Testing" since for me testing implies eg some regular check whether compatibility is still in place

Reply:
That makes sense, I just updated the titles!